_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Micro-procurement for digital work - what's stopping us?"
+ref: working-in-the-open-p1
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at ESDC and TBS"
+date: "2020-07-28"
+excerpt_separator: <!--more-->
+
+---
+
+The Government of Canada (GC) buys a lot of stuff. In 2016, the total spent by the GC on goods and services was ~$15.9B. From an IT point
+of view, this spending can include big ticket items like new data centres and new national applications costing tens of millions of
+dollars, but can also include low dollar value items like paying someone to build a small prototype or write a position paper.
+
+<!--more-->
+
+To ensure that purchasing is working well across the GC, procurement teams are working hard on modernization efforts including the
+development of an [electronic procurement system](https://buyandsell.gc.ca/initiatives-and-programs/the-electronic-procurement-solution/
+about), enabling agile procurement, [social procurement pilots](https://buyandsell.gc.ca/procurement-data/tender-notice/PW-18-00828087)
+and [contract language simplification](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/simpcont-simpcont-eng.html). Interestingly, Low Dollar
+Value (LDV) procurements don't seem to make the list for procurement modernization priorities. This could be because these procurements
+are low risk and generally don't make headline news. However for this same reason they provide the perfect opportunity for procurement and IT experimentation.
```

Comment:
However for this ... 
==>
However, for this ... 

(Missing comma)

Reply:
Fixed, thanks!